src/Bicep.LangServer/Handlers/BicepCompletionHandler.cs

```diff
@@ -0,0 +1,126 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Bicep.Core;
+using Bicep.Core.SemanticModel;
+using Bicep.Core.TypeSystem;
+using Bicep.LanguageServer.CompilationManager;
+using Bicep.LanguageServer.Completions;
+using Bicep.LanguageServer.Utils;
+using OmniSharp.Extensions.LanguageServer.Protocol.Document;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+
+namespace Bicep.LanguageServer.Handlers
+{
+    public class BicepCompletionHandler : CompletionHandler
```

Comment:
It would be good to expand our language server tests. Mind adding some (super basic) tests to validate this functionality?

Reply:
Yeah makes sense. I'll add something specific for this, but I'm also planning to all the missing coverage in a separate PR as well.

Reply:
Sure, sounds good.